modules/client/src/main/java/org/apache/ignite/internal/client/table/ClientRecordBinaryView.java

```diff
@@ -380,4 +392,36 @@ public CompletableFuture<Void> streamData(Publisher<Tuple> publisher, @Nullable
 
         return ClientDataStreamer.streamData(publisher, opts, batchSender, provider, tbl);
     }
+
+    private CompletableFuture<AsyncResultSet<Tuple>> executeAsync(
```

Comment:
let's rename it to more specific

Reply:
The separate method has been removed and all logic has been moved to `queryCriteriaAsync`